extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.Dialog.scss

```diff
@@ -37,3 +37,9 @@
 	margin: 5px auto 0 auto;
 }
 
+.insert-infobox-empty-state {
```

Comment:
Please take a look how similar things are achieved in similar Wikia's feature - maybe empty state in media dialog - and follow the pattern.